thing/index.js

```diff
@@ -372,7 +387,11 @@ function ThingShadowsClient(deviceOptions, thingShadowOptions) {
             // This isn't a Thing topic, so pass it along to the instance if they have
             // indicated they want to handle it.
             //
-            that.emit('message', topic, payload);
+            // Only emit messages that are not also from a potential job topic as well
+            if (!isJobTopic(topicTokens))
```

Comment:
Would it make sense to first check whether the job module is really in use? Otherwise the original handler should apply, shouldn't it?